packages/ra-core/src/auth/useLogout.ts

```diff
@@ -27,20 +26,8 @@ import { LocationDescriptorObject } from 'history';
 const useLogout = (): Logout => {
     const authProvider = useAuthProvider();
     const dispatch = useDispatch();
-
-    /**
-     * We need the current location to pass in the router state
-     * so that the login hook knows where to redirect to as next route after login.
-     *
-     * But if we used useLocation to get it, the logout function
```

Comment:
This is still valid, I don't unbderstrand why you removed it

Reply:
Reverted and updated to the new mechanism